manifests/input.yaml

```diff
@@ -2,778 +2,208 @@ format: yamanifest
-  hashes:
-    binhash: 2785b3ffd59de1fa1e99b7a3d13de8a7
-    md5: 3ea0c260040e47d84c53a917c288cc6e
 work/ocean/INPUT/co2_ryf.nc:
-  fullpath: /g/data/ik11/inputs/access-om2/input_bgc_20220224/mom_025deg/co2_ryf.nc
+  fullpath: /g/data/vk83/experiments/inputs/access-om2/ocean/biogeochemistry/global.025deg/2022.02.24/co2_ryf.nc
   hashes:
-    binhash: 77e3f82a8a92069a66884e946ef59d4e
+    binhash: 95421006948ed72b50bbfe116892ab9d
     md5: c4608d171ce363bc9de8c5d44521d2f4
 work/ocean/INPUT/csiro_bgc.res.nc:
-  fullpath: /g/data/ik11/inputs/access-om2/input_bgc_20220224/mom_025deg/csiro_bgc.res.nc
+  fullpath: /g/data/vk83/experiments/inputs/access-om2/ocean/biogeochemistry/global.025deg/2022.02.24/csiro_bgc.res.nc
   hashes:
-    binhash: d78b86621e3fa9207b5effcb7da1683d
+    binhash: 024cbf10531843523ac730c975197873
     md5: fa1db21962160297748dd514653c1cfa
 work/ocean/INPUT/csiro_bgc_sediment.res.nc:
-  fullpath: /g/data/ik11/inputs/access-om2/input_bgc_20220224/mom_025deg/csiro_bgc_sediment.res.nc
+  fullpath: /g/data/vk83/experiments/inputs/access-om2/ocean/biogeochemistry/global.025deg/2022.02.24/csiro_bgc_sediment.res.nc
   hashes:
-    binhash: c6e07f3e5fefb3b91addd1439c165271
+    binhash: bb9f8342ea72a356d12a97833d02e334
     md5: 539245117b962c3237e4a2a147c2ee39
 work/ocean/INPUT/dust.nc:
-  fullpath: /g/data/ik11/inputs/access-om2/input_bgc_20220224/mom_025deg/dust.nc
+  fullpath: /g/data/vk83/experiments/inputs/access-om2/ocean/biogeochemistry/global.025deg/2022.02.24/dust.nc
   hashes:
-    binhash: 2241179585e814fb22bc8d462a82d4c3
+    binhash: 10ea72e1b6c782f637391b9ec479cbc0
     md5: 64d49862f55fa5d626b37722fa64ba90
 work/ocean/INPUT/grid_spec.nc:
-  fullpath: /g/data/ik11/inputs/access-om2/input_20201102/mom_025deg/grid_spec.nc
+  fullpath: /g/data/vk83/experiments/inputs/access-om2/ocean/grids/mosaic/global.025deg/2020.05.30/grid_spec.nc
   hashes:
     binhash: ae901d0f717e53515ca692138e02f264
     md5: 57663e6575e424cce2cf98bc9778786a
-work/ocean/INPUT/mask_table.465.48x40:
-  fullpath: /g/data/ik11/inputs/access-om2/input_20201102/mom_025deg/mask_table.465.48x40
-  hashes:
-    binhash: 78890536cf18e166ab321c735bfb3d98
-    md5: 10be6631aa435160b4d36531a4f6fd64
 work/ocean/INPUT/ocean_hgrid.nc:
-  fullpath: /g/data/ik11/inputs/access-om2/input_20201102/mom_025deg/ocean_hgrid.nc
+  fullpath: /g/data/vk83/experiments/inputs/access-om2/ocean/grids/mosaic/global.025deg/2020.05.30/ocean_hgrid.nc
   hashes:
     binhash: e9be93a3cf26a195384ee3bc05bd7092
     md5: 38b6f324ae16cc13c180699123ad85b5
 work/ocean/INPUT/ocean_mask.nc:
-  fullpath: /g/data/ik11/inputs/access-om2/input_20201102/mom_025deg/ocean_mask.nc
+  fullpath: /g/data/vk83/experiments/inputs/access-om2/ocean/grids/bathymetry/global.025deg/2023.05.15/ocean_mask.nc
   hashes:
-    binhash: be9a9fee69a77e01905aa46033997d19
-    md5: 238bb01f653ae8a22a7efea4abda5442
+    binhash: 756798128a06a19bd0231ac9170cdc69
+    md5: ee07436042b71df2938726854be5a9aa
```

Comment:
Is this an expected change to `ocean_mask.nc`?

Reply:
This change looks correct

`ee07436042b71df2938726854be5a9aa  /g/data/ik11/inputs/access-om2/input_20230515_025deg_topog/mom_025deg/ocean_mask.nc`